packages/bot/src/scratch/blocks/Logic/controls_if.js

```diff
@@ -31,20 +31,20 @@ Blockly.Blocks.controls_if = {
                     name: 'DO0',
                 },
             ],
-            colour           : Blockly.Colours.Binary.colour,
-            colourSecondary  : Blockly.Colours.Binary.colourSecondary,
-            colourTertiary   : Blockly.Colours.Binary.colourTertiary,
+            colour           : Blockly.Colours.Utility.colour,
+            colourSecondary  : Blockly.Colours.Utility.colourSecondary,
+            colourTertiary   : Blockly.Colours.Utility.colourTertiary,
             // category         : Blockly.Categories.control,
             previousStatement: null,
             nextStatement    : null,
-            tooltip          : translate('If else statement tooltip'),
+            tooltip          : translate('Conditional block'),
             category         : Blockly.Categories.Logic,
         };
     },
     meta(){
         return {
-            'display_name': 'If else statement',
-            'description' : 'If else statement description',
+            'display_name': 'Conditional block',
```

Comment:
Make sure to use the `translate` function for these properties.

Reply:
done